qa/rpc-tests/mempool_reorg.py

```diff
@@ -59,7 +59,8 @@ def run_test(self):
         # Create a block-height-locked transaction which will be invalid after reorg
         timelock_tx = self.nodes[0].createrawtransaction([{"txid": coinbase_txids[0], "vout": 0}], {node0_address: 49.99})
         # Set the time lock
-        timelock_tx = timelock_tx.replace("ffffffff", "11111111", 1)
+        # Disable SequenceLock, Enable nLockTime
+        timelock_tx = timelock_tx.replace("ffffffff", "11111191", 1)
```

Comment:
in case it saves anyone else time maybe add:
`set most signficant bit (little-endian) to activate LOCKTIME_DISABLE_FLAG`